File: BE/org_service1/model/product.go
package model

import "time"

type Product struct {
	ID          string     `json:"id,omitempty"`
	Name        string     `json:"name,omitempty"`
	Avatar      string     `json:"avatar,omitempty"`
	FacilityID  string     `json:"facility_id,omitempty"`
	Description string     `json:"description,omitempty"`
	CreatedAt   time.Time  `json:"created_at,omitempty"`
	UpdatedAt   time.Time  `json:"updated_at,omitempty"`
	DeletedAt   *time.Time `json:"deleted_at,omitempty"`
}

File: syncer/model/product.js
const Sequelize = require('sequelize');
const sequelize = require('../config/pg')
var Product = sequelize.define("product", {
    id: {
        type: Sequelize.BIGINT,
        primaryKey: true,
    },
    name: {
        type: Sequelize.TEXT,
    },
    facilityId: {
        type: Sequelize.BIGINT,
    },
    avatar: {
        type: Sequelize.TEXT,
    },
    description: {
        type: Sequelize.TEXT,
    },
})

module.exports = Product